api/src/main/kotlin/pl/edu/pk/backend/Main.kt

```diff
@@ -21,6 +17,7 @@ fun main() {
   val services = Services(vertx, repositories, Config.appSecret)
   val controllers = Controllers(services)
   val app = App(services, controllers, repositories, database, vertx)
+  InitDatabase(services).initDatabase()
```

Comment:
I think we need more control over this, for example this should only run if some environment variable is present. We don't want this to run on production environment.